Mathlib/Algebra/Group/Basic.lean

```diff
@@ -644,6 +644,14 @@ theorem mul_eq_one_iff_eq_inv : a * b = 1 ↔ a = b⁻¹ :=
 theorem mul_eq_one_iff_inv_eq : a * b = 1 ↔ a⁻¹ = b := by
   rw [mul_eq_one_iff_eq_inv, inv_eq_iff_eq_inv]
 
+@[to_additive]
+theorem mul_eq_one_iff_eq_inv' : a * b = 1 ↔ b = a⁻¹ := by
+  rw [mul_eq_one_iff_inv_eq, eq_comm]
```

Comment:
How do you plan to use these lemmas? Can you just use `rw [mul_eq_one_iff_inv_eq, eq_comm]` instead?

Reply:
This lemma and the one you ask about in the previous comment were both added due to suggestions by Eric, the context is [here](https://github.com/leanprover-community/mathlib4/pull/16032#discussion_r1725944729) and [here](https://github.com/leanprover-community/mathlib4/pull/16032#discussion_r1737282636).

Reply:
OK, let's leave them.